### PR TITLE
fix bug 1188049 - Handle {{bug(123456)}}

### PR DESCRIPTION
**Pull request by jwhitlock**
Fixes about 100 [unknown_kumascript](https://browsercompat.herokuapp.com/importer/issues/unknown_kumascript) issues

### CODE REVIEW

File: mdn/tests/test_kumascript.py [hunk 1]
```diff
@@ -31,6 +31,36 @@ def test_known(self):
...
+        ks = Bug(raw=raw, args=['801425'], scope='footnote')
+        expected = (
+            '<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=801425">'
+            'bug 801425</a>')
+        self.assertEqual(ks.to_html(), expected)
+        self.assertEqual(ks.issues, [])
+
+    def test_summary(self):
+        # https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter
+        raw = "{{bug(1178765, 'summary')}}"
```
Comment (groovecoder) [inline mdn/tests/test_kumascript.py]: This is the kind of test that seems superfluous to me if we already have a test that tests the `min_args` and `max_args` functionality?

**Comment by groovecoder** [comment]
Needs rebase.

File: mdn/tests/test_kumascript.py (hunk 1 above)
Comment (jwhitlock) [inline mdn/tests/test_kumascript.py]: Agreed.  It is supposed to communicate that the two-argument version used on https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter will be treated as an error.  I'll fix that page and drop the test.

**Comment by jwhitlock** [comment]
Rebased, removed extra test

**Pull request merged**